restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -5,13 +5,9 @@
 import com.fasterxml.jackson.core.JsonToken;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ValueNode;
 import com.github.fge.jsonschema.core.exceptions.ProcessingException;
 import com.google.common.base.Splitter;
-import com.google.gson.JsonArray;
```

Comment:
can we also get rid of the dependency or was this transitive of `jsurfer-gson`?

Reply:
Gson was only transitively included via `jsurfer-gson`: I checked that bundles had it before change, but not after.
(Grpc has Gson as a separate dependency for grpc api).